experimental/dds/tree2/src/core/rebase/fence.json

```diff
@@ -1,5 +1,5 @@
 {
 	"tags": ["rebase"],
 	"exports": ["index"],
-	"imports": ["tree", "util", "repair"]
+	"imports": ["tree", "util", "schema-stored"]
```

Comment:
Why do we need the dependency on schema-stored?

Reply:
FieldKey (used by the repair data store) has been moved to schema-stored. This is part of what I'll remove when we no longer need repair data store.

Reply:
Ok. Please update the dependency diagram in the README if need be.